packages/backend-plugin-api/api-report.md

```diff
@@ -107,6 +121,7 @@ export interface BackendPluginRegistrationPoints {
 // @public (undocumented)
 export type BackstageCredentials<TPrincipal = unknown> = {
   $$type: '@backstage/BackstageCredentials';
+  expiresAt?: Date;
```

Comment:
Feels more semantically correct to have it on the credentials right? Thinking that it would always be the underlying minimum expiration that is communicated to the application. Internally the credentials may have a couple of different expirations if it's representing some form of layered credential